index.d.ts

```diff
@@ -68,6 +70,30 @@ export interface Options<Flags extends AnyFlags> {
 	*/
 	readonly flags?: Flags;
 
+	/**
+  Define subcommands.
```

Comment:
Use tab-indentation.

Reply:
The description needs to be more extensive. It should describe how it works (that it doesn't actually call any commands for you, etc).